test/JDBC/expected/BABEL-5242-union-tsql-basetype.out

```diff
@@ -0,0 +1,458 @@
+--
```

Comment:
can we also have test cases like 

select 'unknown'
union
select $1.0
union
select N'xyz'

what would be expected result and result with our code?

Reply:
also this combination,

smallmoney
union
text
union
numeric

Reply:
The above two SQLs should both raise error, since string type cannot be implicitly coerced to numeric type. Supplemented this negative case to test file.